frontend/packages/core/src/Table/stories/metadata-table.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import type { Meta } from "@storybook/react";
+
+import { WizardContext } from "../../Contexts";
+import type { MetadataTableProps } from "../metadata-table";
+import { MetadataTable } from "../metadata-table";
+
+export default {
+  title: "Core/Table/Metadata Table",
+  decorators: [
+    story => (
+      <WizardContext.Provider
+        value={() => {
+          return {
+            onSubmit: () => {},
+            setOnSubmit: () => {},
+            setIsLoading: () => {},
+            displayWarnings: () => {},
+            onBack: () => {},
+          };
+        }}
+      >
+        {story()}
+      </WizardContext.Provider>
+    ),
+  ],
+  component: MetadataTable,
+} as Meta;
+
+const Template = (props: MetadataTableProps) => (
```

Comment:
this is rendering in storybook code preview as `<[object Object] value={() => {}}>` for some reason

Reply:
oh something to do with the decorator?

Reply:
This is the application context wrapper. It looks like the context provider is rendering as the return value (e.g. an object in this case).

Reply:
ok not a blocker